include/flexflow/simulator.h

```diff
@@ -50,11 +50,22 @@ class FFModel;
 
 struct CostMetrics {
   CostMetrics()
-      : forward_time(0.0f), backward_time(0.0f), sync_time(0.0f),
-        memory_requirement(0) {}
+      : forward_time{}, backward_time{}, sync_time{}, inputs_memory{},
```

Comment:
Are these default initializations necessary, or are they going to happen anyway? If they're not necessary let's just get rid of the constructor.